sei-devnet-3/genesis.json

```diff
@@ -0,0 +1,16831 @@
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei13thpswp76ktspfvq0e58csshmyk4tpyzqpnmp9",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1qcmugpu3znmzup5w86xc7u6crh7j7uy83rs07c",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1ejsyjy2qch24sq5qez96tk9vf2a4guvurrk6f8",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1ez2ukd0wtpkptax03wpp5kdx7ns5dgav70lcmr",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei168c6nzwt4ywngf64g6fgafm7w60rcqz94g4tve",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1ekqs8qwljy9t8n7grvcruemegflhaw5thk3rjs",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei150q42nrvdq3x39a68extskmcg83nddlhus5k9r",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1vqq9rvcwsqvepsnp0u457ujtsqrqyh3dh99u9w",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1qv0ghcq442tvtxnztfktpvrccxwfwd3x78qq9g",
+                    "pub_key": null,
+                    "account_number": "0",
+                    "sequence": "0"
+                },
+                {
+                    "@type": "/cosmos.auth.v1beta1.BaseAccount",
+                    "address": "sei1atxjf0tsp8uysf2e3vzptu7qmsqv324flx0zxq",
+                    "pub_key": null,
+   
```

Comment:
there's 1k+ accounts - is that intentional?

Reply:
yeah, for loadtest clients